docs/blog/2019-08-30-building-the-net-third-and-grove-website-in-gatsby/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Building The New Third And Grove Website In Gatsby
+date: 2019-08-30
```

Comment:
BTW, we should update the date to something in the future (same for the filename)

Reply:
thanks, updated